holoviews/plotting/bokeh/__init__.py

```diff
@@ -54,7 +54,7 @@
 
                 # Rasters
                 Image: RasterPlot,
-                GridImage: ImagePlot,
+                GridImage: RasterPlot,
```

Comment:
No, it can be removed, it's just an alias now. Might even make it a function and add a deprecation warning.